src/NationalInstruments.Analyzers/Style/SpellingAnalyzer.cs

```diff
@@ -276,50 +276,6 @@ private static CodeAnalysisDictionary ParseDicDictionary(SourceText text)
         private static string RemovePrefixIfPresent(string prefix, string name)
             => name.StartsWith(prefix, StringComparison.Ordinal) ? name.Substring(1) : name;
 
-        private static IEnumerable<Diagnostic> GetUnmeaningfulIdentifierDiagnostics(ISymbol symbol, string symbolName)
```

Comment:
Unused, I assume?

Reply:
Yes, sorry I didn't comment on that. It was an unused method.